install/configs/karaf/islandora_collection.script

```diff
@@ -1,2 +1,2 @@
-repo-add file:/home/vagrant/islandora/camel/karaf/target/classes/features.xml
-feature:install islandora-collection-service
\ No newline at end of file
+repo-add file:/home/vagrant/Alpaca/karaf/target/classes/features.xml
+feature:install islandora-collection-service
```

Comment:
This the 7.x-2.x camel collection service, we don't use this anymore and so it probably shouldn't be here at all.